prody/proteins/waterbridges.py

```diff
@@ -0,0 +1,1045 @@
+# -*- coding: utf-8 -*-
+
+"""This module detects, predicts and analyzes water bridges.
+"""
+
+__author__ = 'Karolina Mikulska-Ruminska'
+__credits__ = ['Frane Doljanin', 'Karolina Mikulska-Ruminska']
+__email__ = ['karolamik@fizyka.umk.pl', 'fdoljanin@pmfst.hr']
+
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
This needs to be imported in functions not generally, so matplotlib is not a hard a dependency of ProDy